webinar/ally-webinar-Jan2017.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
+<html>
+<head>
+	<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+	<title>Ally – Making course content accesssible</title>
+</head>
+
+<body>
+
+	<div style="Margin:0;padding-top:0;padding-bottom:0;padding-right:0;padding-left:0;min-width:100%;background-color:#eaecec">
```

Comment:
This will be displayed in a browser right? Can we get all these styles in a CSS file?